source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/styles/community.applications.css

```diff
@@ -0,0 +1,2392 @@
+    font-family: fontAwesome;
+  }
+  
+  .featuredIncompatible {
+    color: var(--unraid-66-color);
+  }
+  
+  .greenCardBackground {
+    clip-path: polygon(0 0,100% 0, 100% 100%);
+    background-color: #008100;
+    top: 0px;
+    height: 9rem;
+    width: 9rem;
+    position: relative;
+    left: -10rem;
+    margin-right: -9rem;
+  }
+  
+  .homeMore {
+    color: var(--unraid-66-color);
+    cursor: pointer;
+  }
+  
+  .homeMore:hover {
+    color: #d67777;
+  }
+  
+  .homespotlightIconArea {
+    display: inline-block;
+    position: relative;
+    top: -25rem;
+    left: 24rem;
+    cursor: pointer;
+  }
+  
+  .initDockerSearch {
+    cursor: pointer;
+    text-decoration: none;
+    font-size: 1.5rem;
+  }
+  
+  .initDockerSearch:hover {
+    color: var(--unraid-66-color);
+  }
+  
+  a.initDockerSearch {
+    cursor: pointer;
+    text-decoration: none;
+    color: unset;
+  }
+  
+  .installedCardBackground {
+    clip-path: polygon(0 0,100% 0, 100% 100%);
+    background-color: #322fff;
+    top: 0px;
+    height: 9rem;
+    width: 9rem;
+    position: relative;
+    left: -10rem;
+    margin-right: -9rem;
+  }
+  
+  .installedCardText {
+    text-transform: uppercase;
+    position: absolute;
+    transform: rotate(45deg);
+    -webkit-transform: rotate(45deg);
+    -moz-transform: rotate(45deg);
+    -o-transform: rotate(45deg);
+    color: white;
+    font-size: 2rem;
+    position: absolute;
+    top: 2rem;
+    right: 0rem;
+    width: 6rem;
+    overflow: hidden;
+    height: 2.5rem;
+    display: inline;
+  }
+  
+  input[type=checkbox] {
+    width: 2rem;
+    height: 2rem;
+    margin-left: 0rem;
+  }
+  
+  input[type=button] {
+    background: none;
+    font-size: 1.5rem;
+  }
+  
+  input:hover[type=button] {
+    color: var(--unraid-66-color) !important;
+    background: none !important;
+    background-color: none !important;
+  }
+  
+  input:hover[type=button][disabled] {
+    background: none !important;
+    color: currentColor !important;
+    background-color: none !important;
+  }
+  
+  input[type=button][disabled] {
+    background: none !important;
+    background-color: none !important;
+  }
+  
+  .installedPopup {
+    clip-path: polygon(0 0,100% 0, 100% 100%);
+    background-color: #322fff;
+    top: var(--beta-popup-offset);
+    height: 9rem;
+    width: 9rem;
+    position: absolute;
+    right: 0;
+  }
+  
+  .installedPopupText {
+    position: absolute;
+    transform: rotate(45deg);
+    -webkit-transform: rotate(45deg);
+    -moz-transform: rotate(45deg);
+    -o-transform: rotate(45deg);
+    color: white;
+    font-size: 2rem;
+    position: absolute;
+    top: 2.5rem;
+    right: -1rem;
+    width: 100%;
+    overflow: hidden;
+    height: 2rem;
+  }
+
+  .ca_display_area {
+    display: flex;
+  }
+
+  .mainArea {
+    right: 0px;
+    top: 2rem;
+    display: block;
+    overflow-x: hidden;
+    min-height: 90vh;
+    padding-left: 4.5rem;
+    align-items: stretch;
+  }
```

Comment:
If you use the grid method I mention here - https://github.com/unraid/community.applications/pull/4/files#r2141266907

You could get rid of most of these rules and simplify down to

```css
.mainArea {
    display: block;
    overflow-x: hidden;
}
```